SaltSquareIO/sports-app-react-fe
# Dockerize frontend

[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0fybcp)

## Summary of change

This pull request introduces a Dockerfile and a docker-compose.yml file to enable the application to be built and run inside a Docker container. The main changes include: 
- Added a Dockerfile to define the environment for the app. 
- Created a docker-compose.yml file to streamline the build and run process.

## QA Notes

To test this PR, follow these steps:
1. Ensure you have Docker and Docker Compose installed on your machine.
2. Run the following command to build and start the application:

```bash
docker compose up --build
```

3. Verify that the application is accessible at http://localhost:3000.
